Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM fedora:32
-RUN dnf -y install samba
-COPY tmp/users.tmp /tmp
-COPY tmp/users.sh /tmp/users.sh
-RUN cd /tmp && ./users.sh
-RUN rm /tmp/users.tmp
-CMD /bin/bash -c "smbd --foreground --log-stdout --no-process-group"
+FROM registry.fedoraproject.org/fedora:32
+
+RUN curl -L -o /etc/yum.repos.d/_copr_mskarbek-dinit.repo https://copr.fedorainfracloud.org/coprs/mskarbek/dinit/repo/fedora-32/mskarbek-dinit-fedora-32.repo\
```

Comment:
I'm a little bit worried about including private repos. Is there any other way to use dinit?